ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -4100,7 +4116,22 @@ AHU_CHWSC_HWSC_ECON_EDBPC_RFSS2X_RFVSC2X_SFSS2X_SFVSC2X_SSPC:
   - SFVSC2X
   - SSPC
 
-
+AHU_SS_SFSS_RMM_STM_FDPM_SSPM_SFC_HWRC_EDM_SDM_FSAM_SFVSC_EFVSC:
+  description: "AHU with supply and exhaust pressure, flowrate control, cooling and heating valve control, fire and smoke alarms supply and exhaust rotational velocity sensors"
```

Comment:
You note that this AHU has cooling but I don't see any cooling ABSTRACT types (e.g., CHWSC, DXSC, etc.)